chrome/sunset.js

```diff
@@ -36,32 +36,13 @@ Sunset.start_ = new Date();  // TBD
 
 
 /**
- * The hard deadline date of the deprecation phase.
- *
- * @type {Date}
- * @private
- */
-Sunset.deadline_ = new Date();  // TBD
-
-
-/**
- * The maximum number of days when the extension can be running before
- * it is automatically uninstalled.
- *
- * @type {Number}
- * @private
- */
-Sunset.max_uptime_ = 30;
-
-
-/**
  * Offsets of the notifications in days from the starting date.
  *
  * @type {Array<Number>}
  * @private
  */
 Sunset.timeline_offsets_ = [
-  7,
```

Comment:
The first notification should appear on the first day. No idea why 7 was here. Although we skip the check for notification index = 0, so it doesn't matter anyway.